targets/TARGET_Cypress/scripts/PSOC6.py

```diff
@@ -306,13 +313,29 @@ def complete(message_func, elf0, hexf0, hexf1=None):
 
 def merge_action(args):
     """Entry point for the "merge" CLI command."""
+    try:
+        elf_file = list(pathlib.Path(args.artefacts_location).glob("*.elf"))[0]
+        m4hex_file = list(pathlib.Path(args.artefacts_location).glob("*.hex"))[0]
+    except IndexError:
+        raise ArtefactsError(
+            f"Could not find elf and/or hex file in {args.artefacts_location}"
+        )
+
     complete_func(
-        print, args.elf, args.m4hex, args.m0hex
+        print, elf_file, m4hex_file, args.m0hex
     )
 
 
 def sign_action(args):
     """Entry point for the "sign" CLI command."""
+    try:
+        elf_file = list(pathlib.Path(args.artefacts_location).glob("*.elf"))[0]
+        m4hex_file = list(pathlib.Path(args.artefacts_location).glob("*.hex"))[0]
```

Comment:
@ARMmbed/team-cypress will this work - taking the first bin/hex found in the artifacts ?
I would say yes - there's one application, for one core.